flaskr/manage.py

```diff
@@ -1,14 +1,20 @@
-
-from flask_script import Manager
+# Import packages
+from flask_sqlalchemy import SQLAlchemy
+from flask_script import Manager, Shell
 from flask_migrate import Migrate, MigrateCommand
```

Comment:
so Manager is the ORM?  Migrate Is the migration manager and SQLA is the parent class for the ORM?

Reply:
SQLAlchemy is the ORM, Manager is a library included in Flask-Scripts which acts a lot like Rake.